From: Jeff Mahoney <jeffm@suse.com>
Subject: powerpc: add missing include to book3s_xics.c
Patch-mainline: submitted 5 Jun 2013 to lkml

This fixes the following build failure:
arch/powerpc/kvm/book3s_xics.c:812: error: implicit declaration of function 'get_tb'

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
---
 arch/powerpc/kvm/book3s_xics.c |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/powerpc/kvm/book3s_xics.c
+++ b/arch/powerpc/kvm/book3s_xics.c
@@ -19,6 +19,7 @@
 #include <asm/hvcall.h>
 #include <asm/xics.h>
 #include <asm/debug.h>
+#include <asm/time.h>
 
 #include <linux/debugfs.h>
 #include <linux/seq_file.h>

